docs/modeling-your-data/modeling-your-data-with-dbt/migration-guides/web_to_unified/index.md

```diff
@@ -0,0 +1,577 @@
+alter table  (your_schema)_derived.snowplow_unified_views rename column layout_engine_name_version_major to yauaa__layout_engine_name_version_major;
+alter table  (your_schema)_derived.snowplow_unified_views rename column layout_engine_version to yauaa__layout_engine_version;
+alter table  (your_schema)_derived.snowplow_unified_views rename column layout_engine_version_major to yauaa__layout_engine_version_major;
+alter table  (your_schema)_derived.snowplow_unified_views rename column operating_system_class to yauaa__operating_system_class;
+alter table  (your_schema)_derived.snowplow_unified_views rename column operating_system_name to yauaa__operating_system_name;
+alter table  (your_schema)_derived.snowplow_unified_views rename column operating_system_name_version to yauaa__operating_system_name_version;
+alter table  (your_schema)_derived.snowplow_unified_views rename column operating_system_version to yauaa__operating_system_version;
+alter table  (your_schema)_derived.snowplow_unified_views drop column os_family;
+alter table  (your_schema)_derived.snowplow_unified_views rename column os_major to ua__os_major;
+alter table  (your_schema)_derived.snowplow_unified_views rename column os_minor to ua__os_minor;
+alter table  (your_schema)_derived.snowplow_unified_views rename column os_patch to ua__os_patch;
+alter table  (your_schema)_derived.snowplow_unified_views rename column os_patch_minor to ua__os_patch_minor;
+alter table  (your_schema)_derived.snowplow_unified_views rename column os_version to ua__os_version;
+alter table  (your_schema)_derived.snowplow_unified_views rename column page_views_in_session to views_in_session;
+alter table  (your_schema)_derived.snowplow_unified_views rename column page_view_id to view_id;
+alter table  (your_schema)_derived.snowplow_unified_views rename column page_view_in_session_index to view_in_session_index;
+alter table  (your_schema)_derived.snowplow_unified_views rename column primary_impact to iab__primary_impact;
+alter table  (your_schema)_derived.snowplow_unified_views rename column reason to iab__reason;
+alter table  (your_schema)_derived.snowplow_unified_views rename column spider_or_robot to iab__spider_or_robot;
+alter table  (your_schema)_derived.snowplow_unified_views rename column useragent_family to ua__useragent_family;
+alter table  (your_schema)_derived.snowplow_unified_views rename column useragent_major to ua__useragent_major;
+alter table  (your_schema)_derived.snowplow_unified_views rename column useragent_minor to ua__useragent_minor;
+alter table  (your_schema)_derived.snowplow_unified_views rename column useragent_patch to ua__useragent_patch;
+alter table  (your_schema)_derived.snowplow_unified_views rename column useragent_version to ua__useragent_version;
+alter table  (your_schema)_derived.snowplow_unified_views add column if not exists default_channel_group varchar(25);
+alter table  (your_schema)_derived.snowplow_unified_views add column event_name varchar(1000);
```

Comment:
Do all warehouses allow you to say `varchar` types? In particular bigquery can be fussy? Maybe just call it out as a note at the top rather than add loads of different versions